tutorial/connect-from-aks/ConnectFromAKS/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Azure.Core;
+using Azure.Identity;
+using StackExchange.Redis;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace ConnectFromAKSSample
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            try
+            {
+                string connectionOption = Environment.GetEnvironmentVariable("CONNECTION_OPTION");
+                string cacheHostName = Environment.GetEnvironmentVariable("CACHE_HOSTNAME");
+                ConfigurationOptions? configurationOptions = null;
+
+                switch (connectionOption)
+                {
+                    case "MANAGED_IDENTITY":
+                        Console.WriteLine($"Connecting with managed identity..");
+                        configurationOptions = await ConfigurationOptions.Parse($"{cacheHostName}:6380").ConfigureForAzureWithTokenCredentialAsync(new DefaultAzureCredential());
+                        configurationOptions.AbortOnConnectFail = false; // Fail fast for the purposes of this sample. In production code, this should remain false to retry connections on startup
+                        break;
+
+                    case "ACCESS_KEY":
+                        Console.WriteLine("Connecting with an access key.");
+                        string connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING");
+                        configurationOptions = ConfigurationOptions.Parse(connectionString);
+                        configurationOptions.AbortOnConnectFail = true; // Fail fast for the purposes of this sample. In production code, this should remain false to retry connections on startup
+                        break;
+
+                    default:
+                        return;
+
+                }
+
+                // Create a connection to Redis
+                ConnectionMultiplexer redis = await ConnectionMultiplexer.ConnectAsync(configurationOptions);
```

Comment:
start this line with `using` so the `ConnectionMultiplexer` gets disposed properly